optionals/no_missing_mod_check/script_component.hpp

```diff
@@ -0,0 +1,16 @@
+#define COMPONENT no_missing_mod_check
+#include "\x\cba\addons\main\script_mod.hpp"
+
+
+#ifdef DEBUG_ENABLED_CACHE_DISABLE
+	#define DEBUG_MODE_FULL
+#endif
+
+#ifdef DEBUG_SETTINGS_CACHE_DISABLE
+	#define DEBUG_SETTINGS DEBUG_SETTINGS_CACHE_DISABLE
+#endif
+
+#undef REQUIRED_VERSION
+#define REQUIRED_VERSION 1.00
```

Comment:
Pointless, this optional requires 2 other components which have required version set from `main\script_mod.hpp` anyways.

Reply:
Same for cache disable then